packages/react-card/Spec.md

```diff
@@ -0,0 +1,432 @@
+
+2. Hover
+
+   Cursor style changes to `pointer`, background color and elevation follows interactive styling to show visual feedback
+
+3. Pressed
+
+   Cursor does not change background color and elevation follows interactive styling to show visual feedback for pressed state.
+
+4. onClick
+
+   Card and Checkbox are both in selected state, clicking again toggles components to unselected state
+
+### Non-interactive cards
+
+![Visual interaction of non-interactive cards using the mouse](./assets/non-interactive-interaction-mouse.png)
+
+Cards that have multiple actionable child components are expected to not be clickable. We preserve the use of `EnterKey` for moving focus to actionable controls inside the card, therefore the card surface should not be used to set the state of the card but is reliant on the checkbox or similar inputs to set the card state.
+
+If users have a scenario that requires them to use actionable components within a clickable Card then the bubbling of events is expected to be managed by the users themselves.
+
+Mouse interaction
+
+- OnMouseEnter / Hover
+
+  Cursor style stays at default, rest state stays persistent no visual feedback is applied to the card surface
+
+- Hover & onClick (Checkbox)
+
+  Hover state & pressed state applies to checkbox component
+
+- onClick
+
+  (Checkbox) Card and Checkbox are both in selected state, clicking again toggles components to unselected state
+
+#### Disabled
+
+![Visual interaction of Disabled cards using the mouse](./assets/disabled.png)
+
+1. Unselected disabled
+
+   - Label: Selectable Card
+   - Role: Group
+   - State: Disabled, unselected
+
+2. Selected disabled
+
+   - Label: Selectable Card
+   - Role: Group
+   - State: Disabled, selected
+
+### Keyboard
+
+### Interactive cards
+
+![Visual interaction of interactive cards using the keyboard](./assets/interactive-interaction-keyboard.png)
+
+1. Rest
+
+2. Keyboard interaction
+
+   EnterKey Perfoms action on card “Open file, Open link in browser etc...”
+
+#### Selectable cards
+
+![Visual interaction of selectable cards using the keyboard](./assets/selectable-interaction-keyboard.png)
+
+1. Rest Default state is ‘unselected’
+2. Focus Focus rectangle is shown on card
+3. Focus Use spacebar to toggle state (selected/unselected)
+
+### Non-interactive cards
+
+![Visual interaction of non-interactive cards using the keyboard](./assets/non-interactive-interaction-keyboard.png)
+
+Keyboard interaction
+
+1. Focus
+
+   Initial focus is set on card, to access items within the card use EnterKey or Spacebar
+
+2. Focus-Inside
+
+   Focus moves to first actionable item in the card. Use ArrowKeys or TabKey to move focus to child elements within the card. To exit card use EscKey, to move focus back to card [parent]
+
+3. Checkbox-Focus
+
+   Depending on your tab order of items within the card, the number of keystrokes will vary. Once focus is set on the checkbox by default it will be set to unselected
```

Comment:
Does this mean that if I focus a selected checkbox it will be unselected ? 👀

Reply:
Poor phrasing in the design spec, supposed to say that by default the checkbox is unselected and not that it will be unselected when focusing. Removed that altogether.